src/components/DragDrop/DragDrop.tsx

```diff
@@ -96,7 +99,8 @@ export const DragDrop = ({
       );
       childrenWidth += item.clientWidth + itemMarginLeft;
     });
-    if (overflowListItems) {
```

Comment:
`overflowListItems` is always truthy (since it's an array), and we don't need this conditional. We do, however, now need to do a null check on `dragDropRef.current`.